src/components/login/Login.js

```diff
@@ -2,19 +2,26 @@ import React, { useEffect } from "react";
 import styled from "styled-components";
 import { useNavigate } from "react-router-dom";
 import axios from "axios";
+import { USER_TYPE } from "../../constants/common";
 
 const Login = () => {
+  const handleStorage = () => {
```

Comment:
아래처럼 함수명을 바꾸는 것은 어떨까요?

```suggestion
const removeTokenStorage = () => {
```